x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step.tsx

```diff
@@ -19,14 +19,19 @@ export const AdvancedStep: FC<CreateAnalyticsStepProps> = ({
   setCurrentStep,
   stepActivated,
 }) => {
+  const showForm = step === ANALYTICS_STEPS.ADVANCED;
+  const showDetails = step !== ANALYTICS_STEPS.ADVANCED && stepActivated === true;
+
+  const dataTestSubj = `mlAnalyticsCreateJobWizardAdvancedStep ${showForm ? 'active' : ''}${
+    showDetails ? 'summary' : ''
```

Comment:
I'd suggest to move the space character to the optional labels, otherwise we have a trailing space in case the step was not active yet:
```
  const dataTestSubj = `mlAnalyticsCreateJobWizardAdvancedStep${showForm ? ' active' : ''}${
    showDetails ? ' summary' : ''
```